frontend/src/metabase/dashboard/actions/utils.unit.spec.ts

```diff
@@ -135,14 +136,18 @@ describe("dashboard > actions > utils", () => {
       const oldCards = [
         createMockDashboardOrderedCard({
           id: 1,
-          visualization_settings: { foo: { bar: { baz: 21 } } },
+          visualization_settings: {
+            foo: { bar: { baz: 21 } },
+          } as VisualizationSettings,
         }),
         createMockDashboardOrderedCard({ id: 2 }),
       ];
       const newCards = [
         createMockDashboardOrderedCard({
           id: 1,
-          visualization_settings: { foo: { bar: { baz: 22 } } },
+          visualization_settings: {
+            foo: { bar: { baz: 22 } },
+          } as VisualizationSettings,
```

Comment:
Can we avoid type casting? Probably we could use `createMockVisualizationSettings`